Detectors/TPC/calibration/SpacePoints/include/SpacePoints/TrackResiduals.h

```diff
@@ -299,12 +299,12 @@ class TrackResiduals
   /// \param ip Bin index in Y/X
   /// \param iz Bin index in Z/X
   /// \return global bin number
-  unsigned short getGlbVoxBin(int ix, int ip, int iz) const;
+  size_t getGlbVoxBin(int ix, int ip, int iz) const;
```

Comment:
`unsigned int` should have been sufficient, but I doubt that it has an impact to stay with `size_t`